docs/development.md

```diff
@@ -0,0 +1,74 @@
+# Setting Up a Development Environment
+
+
+## Table of contents
+
+1. [Installing the Zendesk App Tools (ZAT)](#dev-zat)
```

Comment:
In markdown you get a numbered list by prefixing all lines with `1. `. so if you need to insert another item you don't need to re-number. 

eg:

1. a
1. b
1. c

Reply:
Cool! I learned something new today!